lib/main_common.pm

```diff
@@ -1099,6 +1099,7 @@ sub load_consoletests {
     loadtest "console/system_state";
     loadtest "console/prepare_test_data";
     loadtest "console/consoletest_setup";
+    loadtest "console/systemd_wo_udev" unless is_sle('<15-sp4') && is_leap('<15.4');
```

Comment:
```suggestion
    loadtest "console/systemd_wo_udev" unless is_sle('<15-sp4') || is_leap('<15.4');
```

Reply:
oh yes, good one :) 
Btw, should we enable this also for TW?

Reply:
The dep has been removed in factory/TW since v248,4. So yes you can enable this for TW as well.